setup.py

```diff
@@ -13,8 +13,10 @@
     long_description_content_type="text/markdown",
     url="https://github.com/lc5415/raceplotly",
     packages=setuptools.find_packages(),
+    setup_requires=["numpy"],
```

Comment:
Not needed, really, pandas already has numpy as a dependency, what I'll do instead is to remove the numpy dependency and this line and bump the minimum python requirement to 3.7 as well as change the tests